Gemfile

```diff
@@ -8,8 +8,7 @@ group :development do
   gem 'compass', '1.0.3'
   gem 'sass-globbing', '1.1.5'
   gem 'stringex', '2.8.6'
-  # > 2.1.0 causes slowdowns https://github.com/sass/sassc-ruby/issues/189
-  gem 'sassc', '2.1.0'
+  gem 'sassc', '2.4.0'
```

Comment:
We cannot change this, as per the comment on the existing code.

See: https://github.com/sass/sassc-ruby/issues/189

If this is problematic for Windows, I think it is better to start documenting you cannot use this on Windows.

../Frenck

Reply:
Oh, I completely misread the purpose of that comment! I'll try to sort this out with the older version - see if there's some other way to make that gem compatible to be installed at 2.1.0